components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -1247,12 +1253,13 @@ class NewTabPage extends React.Component<Props, State> {
               showPhotoInfo={!isShowingBrandedWallpaper && hasWallpaperInfo && newTabData.showBackgroundImage}
               onClickSettings={this.openSettings}
               onDismissBraveTalkPrompt={this.props.actions.dismissBraveTalkPrompt}
+              color={overriddenForegroundColor ?? '#ffffff'}
             />
             </Page.FooterContent>
           </Page.Footer>
           {newTabData.showToday &&
           <Page.GridItemNavigationBraveToday>
-            <BraveTodayHint />
+            <BraveTodayHint color={overriddenForegroundColor ?? 'white'}/>
```

Comment:
Nit: Make this use a hex color, like the other ones
```suggestion
            <BraveTodayHint color={overriddenForegroundColor ?? '#ffffff'}/>
```